tests/examples/hello/hello/hello.py

```diff
@@ -0,0 +1,167 @@
+import struct
+from enum import IntEnum
+from typing import Any
+
+import numpy as np
+
+import legate.core.types as types
+from legate.core import Rect, Store, get_legate_runtime
+
+from .library import user_context, user_lib
+
```

Comment:
Should we also include the "driver" code, which does the actual calls to print_hello, iota etc.?

Reply:
Yep, I missed that on the first commit. Added.